### PR TITLE
css-three-column-layout

### PR DESCRIPTION
**Pull request by Alex-piper**
Screen Shot 2022-10-14 at 1 26 11 PM

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  text-align: center;
+}
+
+h2 {
+  font-size: 24px;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+  display: grid;
```
Comment (Zhiwutian) [inline css-three-column-layout/styles.css:34]: this should be flex instead of grid

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  text-align: center;
+  margin: 10px;
+}
+
+img {
+  width: 40px;
+}
+
+.white-circle {
+  display: grid;
```
Comment (Zhiwutian) [inline css-three-column-layout/styles.css:56]: same as above comment.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
just a small update on the display properties

**Review by Zhiwutian — APPROVED** [review]
Great work Alex!

**Review by Alex-piper — COMMENTED** [review]
updated

**Pull request merged**